src/shared/trace.js

```diff
@@ -81,7 +82,6 @@ export const trace = (
          * We disable this lint rule as the linter does not appear to
          * understand the optional chaining.
          */
-        // eslint-disable-next-line flowtype/no-unused-expressions
```

Comment:
huh - what fixed this?

Reply:
I think the update to eslint flowtype plugin? It was complaining when I tried to commit until I removed these lines.